ui/lib/apps/KeyViz/heatmap/index.tsx

```diff
@@ -2,15 +2,15 @@ import React, { useRef, useEffect } from 'react'
 import * as d3 from 'd3'
 import { useEventListener } from 'ahooks'
 import { heatmapChart } from './chart'
-import { HeatmapData, DataTag, HeatmapRange } from './types'
+// import { HeatmapData, DataTag, HeatmapRange } from './types'
 
-type HeatmapProps = {
-  data: HeatmapData
-  dataTag: DataTag
-  onBrush: (selection: HeatmapRange) => void
-  onZoom: () => void
-  onChartInit: (any) => void
-}
+// type HeatmapProps = {
+//   data: HeatmapData
+//   dataTag: DataTag
+//   onBrush: (selection: HeatmapRange) => void
+//   onZoom: () => void
+//   onChartInit: (any) => void
```

Comment:
Why do we have these changes?

Reply:
This type is unused (reported by eslint), we can remove it.

Reply:
Done.